genomesubsampler/prototypeSelection.py

```diff
@@ -62,11 +109,10 @@ def distance_sum(elements, dm):
     function signature with type annotation for future use with python >= 3.5
     def distance_sum(elements: Sequence[str], dm: DistanceMatrix) -> float:
     '''
-
     return np.tril(dm.filter(elements).data).sum()
 
 
-def prototype_selection_exhaustive(dm, num_prototypes,
+def prototype_selection_exhaustive(dm, num_prototypes, seedset=[],
```

Comment:
Thanks for explaining this! This is indeed something I can't learn from Google (since I didn't run into error I won't Google). I have changed all the instances accordingly.